### PR TITLE
Bug fix redirects from write / draw step to next steps to matchup to scoring

### PR DESCRIPTION
**Pull request by rhiannonstanford**
This PR fixes incorrect redirects.  Ensure that the game flow goes from write / draw step / submission to next steps to matchup to pointshare.  

These changes are made in the Gameplay directory.  This is the most current code and it's components are currently being routed to and used.

### CODE REVIEW

File: src/components/pages/GamePlay/GameWriteStep.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ export default function GameWriteStep(props) {
 
       setIsUploading(false);
     }, triggerSubmitTimer);
+    history.push('/child/next-steps');
```
Comment (rhiannonstanford) [inline src/components/pages/GamePlay/GameWriteStep.js:52]: After story / drawing submission, page routes to Next Steps page.

File: src/components/pages/GamePlay/GameWriteStep.js [hunk 2]
```diff
@@ -150,11 +151,12 @@ export default function GameWriteStep(props) {
                 : "I'd rather draw"}
             </button>
 
-            {props.battleReady && (
+            {/* This button is no longer needed as the page routes to next steps. If in the future a battle mode is implemented, this button will be used. */}
```
Comment (rhiannonstanford) [inline src/components/pages/GamePlay/GameWriteStep.js:154]: This button is no longer needed as page auto routes to Next Steps with closing of modal.

File: src/components/pages/MatchUp/RenderMatchUp.js [hunk 3]
```diff
@@ -13,8 +13,8 @@ const RenderMatchUp = props => {
   const [faceoffs, setFaceoffs] = useState([]);
   const [modalVisible, setModalVisible] = useState(true);
 
-  const goToChildDashboard = () => {
-    push('/child/dashboard');
+  const goToPointShare = () => {
```
Comment (rhiannonstanford) [inline src/components/pages/MatchUp/RenderMatchUp.js:16]: Matchup should go to PointShare page when complete.

File: src/components/pages/NextSteps/RenderNextSteps.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ const RenderNextSteps = () => {
   const { push } = useHistory();
 
   const goToMatchup = () => {
-    push('/gameplay/mission/matchup');
+    push('/child/match-up');
```
Comment (rhiannonstanford) [inline src/components/pages/NextSteps/RenderNextSteps.js:11]: Next Steps should go to Match up page when complete.

**Review by rhiannonstanford — COMMENTED** [review]
Self review.

**Review by Yonathan-Admasu728 — APPROVED** [review]
Excellent work.  finally those routes are on point.

**Review by artofmayhem — COMMENTED** [review]
Nice job connection this all up. We should go over tomorrow

**Review by artofmayhem — APPROVED** [review]

**Pull request merged**